rust/Cargo.lock

```diff
@@ -85,9 +88,9 @@ dependencies = [
 
 [[package]]
 name = "ed25519"
-version = "1.3.0"
+version = "1.2.0"
```

Comment:
@satran004 this is what was bombing out. I needed a cargo update to generate the lib